game/scripts/vscripts/items/satanic_core.lua

```diff
@@ -124,3 +125,23 @@ function modifier_item_satanic_core:OnTakeDamage( kv )
 end
 
 --------------------------------------------------------------------------------
+
+modifier_satanic_core_unholy = class({})
+
+function modifier_satanic_core_unholy:DeclareFunctions()
+  return {
+    MODIFIER_PROPERTY_TOOLTIP
```

Comment:
`modifier_item_satanic_core` increased lifesteal based on the presence of the active modifier. So if it did add lifesteal it was causing double lifesteal anyway.